.github/workflows/test-and-deploy.yml

```diff
@@ -37,20 +37,18 @@ jobs:
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
-      # We use 18.19.0 because it is the first backport to support --import
-      # https://raw.githubusercontent.com/nodejs/node/main/doc/changelogs/CHANGELOG_V18.md
-      - name: Use Node.js 18.19.0
+      - name: Use Node.js 20.x
         uses: actions/setup-node@v4
         with:
-          node-version: 18.19.0
+          node-version: 20.x
```

Comment:
I would prefer a fixed version here, even with minor versions nodejs tends to introduce breaking changes and it would be good to detect those situations.

Reply:
The same applies to the version that we use for `build_dist`, really, that way we can detect early of when we should upgrade the versions on the AMI.

Reply:
(note: these things have happened in the past, this is not just me being paranoid :-D)

Reply:
Agreed, its better to tie a specific version

Reply:
Will tie to a specific version